xbmc/favourites/FavouritesService.cpp

```diff
@@ -77,8 +77,16 @@ static bool LoadFromFile(const std::string& strPath, CFileItemList& items)
   return true;
 }
 
-CFavouritesService::CFavouritesService(std::string userDataFolder) : m_userDataFolder(std::move(userDataFolder))
+CFavouritesService::CFavouritesService(std::string userDataFolder)
```

Comment:
would it make sense to change parameter type to <code>const std::string &</code> as you're no longer using the path-by-value-then-move idiom here? Alternatively, you could use <code>std::move</code> to assign <code>m_userDataFolder</code> in <code>CFavouritesService::ReInit</code>.